pom.xml

```diff
@@ -44,6 +49,32 @@
 				<groupId>org.springframework.boot</groupId>
 				<artifactId>spring-boot-maven-plugin</artifactId>
 			</plugin>
+			<plugin>
+				<groupId>org.jacoco</groupId>
+				<artifactId>jacoco-maven-plugin</artifactId>
+				<version>0.8.7</version>
+				<configuration>
+					<excludes>
+						<exclude>org/dcsa/bkg/config/*</exclude>
+						<exclude>org/dcsa/bkg/model/**</exclude>
```

Comment:
How come we should exclude models? We sometimes have logic inside models (e.g., some of the enums or the OperationsEvent in OVS)